service/service.go

```diff
@@ -212,8 +212,7 @@ func (srv *Service) initExtensionsAndPipeline(ctx context.Context, set Settings,
 	}
 
 	if cfg.Telemetry.Metrics.Level != configtelemetry.LevelNone && cfg.Telemetry.Metrics.Address != "" {
-		// The process telemetry initialization requires the ballast size, which is available after the extensions are initialized.
```

Comment:
Was this comment removed intentionally?

Reply:
That was an accident thanks for catching that!